### PR TITLE
add executor comment

### PR DESCRIPTION
**Pull request by nevermore3**
## What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Comment by Sophie-Xie** [comment]
Will you modify log in this PR?

**Comment by nevermore3** [comment]
> 

yes

**Comment by nevermore3** [comment]
> Will you modify log in this PR?

yes

File: src/graph/executor/Executor.cpp [hunk 1]
```diff
@@ -1,7 +1,6 @@
-/* Copyright (c) 2020 vesoft inc. All rights reserved.
- *
- * This source code is licensed under Apache 2.0 License.
- */
+// Copyright (c) 2022 vesoft inc. All rights reserved.
```
Comment (darionyaphet) [inline src/graph/executor/Executor.cpp:1]: maybe should not update the year info. It represent the file create time and maybe make reader confuse.

File: src/graph/executor/admin/SubmitJobExecutor.cpp [hunk 2]
```diff
@@ -99,7 +96,7 @@ StatusOr<DataSet> SubmitJobExecutor::buildResult(meta::cpp2::JobOp jobOp,
     }
       // no default so the compiler will warning when lack
   }
-  DLOG(FATAL) << "Unknown job operation " << static_cast<int>(jobOp);
```
Comment (Shylock-Hg) [inline src/graph/executor/admin/SubmitJobExecutor.cpp:102]: Don't change this.

File: src/graph/executor/Executor.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/graph/executor/Executor.cpp:1]: Agree.

Comment (nevermore3) [reply src/graph/executor/Executor.cpp]: done

File: src/graph/executor/admin/SubmitJobExecutor.cpp (hunk 2 above)
Comment (nevermore3) [reply src/graph/executor/admin/SubmitJobExecutor.cpp]: fix

**Review by Shylock-Hg — APPROVED** [review]
LGTM

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**